configs/prow-dev/config/external_plugins_config.yaml

```diff
@@ -15,6 +15,15 @@ ti-community-owners:
       - tidb-community-bots/test-dev
     default_require_lgtm: 1
     sig_endpoint: https://bots.tidb.io/ti-community-bot
+    trusted_teams:
```

Comment:
Revert these changes because we need  release it first.

Reply:
ok